gusto/state.py

```diff
@@ -194,6 +195,7 @@ def __init__(self, mesh, vertical_degree=None, horizontal_degree=1,
 
         self.Omega = Coriolis
         self.mu = sponge_function
+        self.h = hydrostatic
```

Comment:
`h` feels like a very bad name for this attribute.  A naive reader might think that `h` is somehow related to mesh spacing?  Why not just call it `hydrostatic`?